pages/common/git-credential-cache.md

```diff
@@ -0,0 +1,8 @@
+# git credential-cache
+
+> `git` helper to temporarily store passwords in memory.
```

Comment:
```suggestion
> Git helper to temporarily store passwords in memory.
```

Reply:
I can't apply them automatically: no such button available for me. :( So, please apply them yourself if u think it makes PR better.